Sources/Plasma/PubUtilLib/plInputCore/plAvatarInputInterface.cpp

```diff
@@ -308,7 +313,7 @@ void    plAvatarInputInterface::RestoreDefaultKeyMappings()
     fControlMap->BindKey( KEY_NUMPAD_SUBTRACT,          B_CAMERA_ZOOM_OUT );
 
     fControlMap->BindKey( KEY_SHIFT,                    B_CONTROL_MODIFIER_FAST );
-    fControlMap->BindKey( KEY_Z,                        B_CONTROL_MODIFIER_STRAFE );
+    fControlMap->BindKey( KEY_ALT,                      B_CONTROL_MODIFIER_STRAFE );
```

Comment:
Ah, the infamous Z bug...  I guess people can stop calling me "rax" now ;)

Reply:
:trollface: